src/services/TimeSrv.ts

```diff
@@ -378,7 +378,9 @@ export class TimeSrv {
   };
 
   timeRange(): TimeRange {
-    return getTimeRange(this.time, this.timeModel);
+    // @ts-ignore default time
+    const time = getTemplateSrv()?.timeRange?.raw || getDefaultTimeRange().raw;
+    return getTimeRange(time, this.timeModel)
```

Comment:
Is it possible to reassign `this.time` to avoid using `// @ts-ignore default time`?

Reply:
updated